contracts/ownership/Ownable.sol

```diff
@@ -42,6 +42,9 @@ contract Ownable {
 
   /**
    * @dev Allows the current owner to relinquish control of the contract.
+   * @notice Renouncing to ownership will leave the contract without an owner.
+   * It will not be possible to call anymore the functions with the onlyOwner
```

Comment:
nitpick: the english would flow better like:

"It will not be possible to call the functions with the onlyOwner modifier anymore."